docs/Processes.md

```diff
@@ -1435,3 +1435,200 @@ and are under the `.ctp` namespace.
  | schema               |    retrieve schema from tickerplant    |                                                                   `1b`|
  | clearlogonsubscription  | clear log on subscription, only called if createlogfile is also enabled  |                              `0b`|
 
+TorQ Data Quality System Architecture
+-------
+
+The Data Quality System consists of two processes that are listed below: 
+Data Quality Checker (DQC) and Data Quality Engine(DQE). The Purpose of 
+the Data Quality System is to ensure quality of data in the data capturing
+system by running checks on other TorQ processes. The system behaves based
+on Data Quality Config files. The metrics from the config files and the
+results of the checks performed on databases in the data capturing system
```

Comment:
There is no mention of check functionality being in the code base and it being leveraged by the configuration.  This would allow you to mention the fact that checks are performed once or multiple times and between two times for example.